controllers/src/userController/Controllers.cpp

```diff
@@ -36,6 +37,13 @@ int Controllers::getFormatedFreq()
   return round10(freq);
 }
 
+void Controllers::displayFreq() {
+  char buffer[6];
+  int freq = Controllers::getFormatedFreq();
+  sprintf(buffer, "%3d.%02d", freq / 100, freq % 100);
```

Comment:
Hmm What about simple divide freq by 100 and assign it to float or better, char? There would always be 2 decimal places(i'm not sure now)

Is it possible to add immediately ` MHz` in sprintf string?

Reply:
### g++
```
int getFormatedFreq() {
    return 8890;
}

int main() {
    char buffer[11];
    int i = getFormatedFreq();
    float f = (float) i / 100;
    sprintf(buffer, "%6.2f MHz", f);
    printf("%s\n", buffer);
    return 0;
}
```
Result
```
 88.90 MHz
```
### Arduino
```
void Controllers::displayFreq() {
  char buffer[11];
  int freq = Controllers::getFormatedFreq();
  float f = (float) freq / 100;
  sprintf(buffer, "%6.2f", f);
  Lcd::write(0, 0, buffer);
}
```
Result
```
     ?
````
I'm going to leave it with ```freq / 100```;```freq % 100``` but add ```MHz``` to buffer.

Reply:
😆  okej